src/test/resources/sample.servlet/testLooseApplicationWithWarTask.gradle

```diff
@@ -48,40 +48,32 @@ repositories {
     mavenCentral()
 }
 
-configurations {
-  war2.extendsFrom runtime
-}
-
 dependencies {
     testCompile 'junit:junit:4.12'
     providedCompile group: 'javax.servlet', name: 'javax.servlet-api', version:'3.1.0'
     compile 'org.apache.commons:commons-text:1.1'
-    war2 'org.apache.commons:commons-proxy:1.0'
     libertyRuntime group: runtimeGroup, name: runtimeArtifactId, version: runtimeVersion
 }
 
-task war1(type:War){
+war {
     baseName = 'sample.servlet1'
 }
 
-task war2(type:War){
-    baseName = 'sample.servlet2'
-    classpath = configurations.war2
-}
-
 liberty {
     server{
         name = wlpServerName
         looseApplication = true
         stripVersion = true
-        serverXmlFile = file("src/main/liberty/config/server-list-test.xml")
+        serverXmlFile = file("src/main/liberty/config/server-apps-test.xml")
         packageLiberty {
             packageDirectory = packageDir
             packageName = appName
             include = packagingType
         }
-    	apps = [war1, war2]
-      verifyAppStartTimeout = 30
+        deploy {
+            apps = [war]
```

Comment:
Is this still testing two apps? I don't really understand the changes here. And I see a different serverXmlFile referenced above.

Reply:
No it's not. This test wasn't configured correctly. The server.xml was wrong and the tests were checking for one app.